include/roboteam_ai/stp/computations/PassComputations.h

```diff
@@ -28,6 +28,7 @@ class PassComputations {
      * @param ballLocation location of the ball
```

Comment:
Could you write the function description more clear? It looks to me like the function wants to pass to the robots instead of other points in the field